pipeline/io.py

```diff
@@ -0,0 +1,99 @@
+import base64
+
+
+class File:
+    """
+    IO file descriptor. Allows to manipulate files between client and server.
+
+    Example: (Transfer an audio file)
+
+        '''
+        On client
+        '''
+
+        from pipeline.io import File
+
+        FILE_PATH = "files/audio.flac"
+
+        # Convert file into a string
+        input_data = File(FILE_PATH).to_api()
+
+        run = client.run_pipeline("pipeline_ID", [input_data])
+
+        '''
+        On server
+        '''
+        from pipeline.io import File
+        import tempfile
+
+        file = File()
+        file_as_bytes = file.file_get_bytes(input_data)
+        temp_file = tempfile.NamedTemporaryFile(delete=True, mode="wb", suffix=file.extension, dir="")
+        temp_file.write(file_as_bytes)
+
+        # Do whatever to the file
+        # ...
+        # Close and delete file
+        temp_file.close()
```

Comment:
Am I right in thinking that the user should use code like this inside their Pipeline functions?

Reply:
That's right! In the case of Whisper's model, the input expected is an audio file (hence the example with a temp file). The user is expected to re-use this File class inside pipeline function to manipulate the file in any desired format.